cfme/tests/services/test_service_catalogs_bundles.py

```diff
@@ -0,0 +1,142 @@
+import fauxfactory
+import pytest
+from widgetastic.utils import partial_match
+
+from cfme import test_requirements
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.services.myservice import MyService
+from cfme.services.service_catalogs import ServiceCatalogs
+from cfme.utils.generators import random_vm_name
+from cfme.utils.wait import wait_for
+
+
+pytestmark = [
+    pytest.mark.meta(server_roles="+automate"),
+    pytest.mark.long_running,
+    test_requirements.service,
+    pytest.mark.tier(2),
+    pytest.mark.provider([VMwareProvider], selector=ONE_PER_TYPE, scope="module"),
+]
+
+
+@pytest.fixture(scope="function")
+def catalog_item_setups(appliance, provider, provisioning, request, catalog, dialog):
```

Comment:
The test is failing in PRT because the provider is not set up in the appliance before catalog item creation. Add the setup_provider_modscope fixture:

```
def catalog_item_setups(appliance, provider, setup_provider_modscope, provisioning, request, catalog, dialog):
```

Reply:
Thanks for pointing the issue